### PR TITLE
Create score_update_logic.py

### PR DESCRIPTION
**Pull request by harshitashyale**
Updating Score Logic

### CODE REVIEW

File: score_update_logic.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+def sum(a, b, c):
+    return a + b + c
+
+
+def calculate_score(playerX, playerY):
```
Comment (harshitashyale) [inline score_update_logic.py:5]: There is a better way to do this?

**Comment by DurvaG8** [comment]
do you think there’s a better way to do this?

**Comment by harshitashyale** [comment]
Yes Maybe

**Comment by harshitashyale** [comment]
Do you have any suggestions - I'm stuck

**Comment by DurvaG8** [comment]
How about if we try to redo it?

File: score_update_logic.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        for move in moves_Y:
+            playerY[move] = 1
+
+        score_X, score_Y = calculate_score(playerX, playerY)
+        print("Score for Player X:", score_X)
+        print("Score for Player Y:", score_Y)
+
+        play_again = input("Do you want to calculate score again? (yes/no): ")
+        if play_again.lower() != "yes":
+            break
```
Comment (DurvaG8) [inline score_update_logic.py:42]: Can we do this in any other way?

File: score_update_logic.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        moves_Y = list(map(int, input("Moves for Y: ").split()))
+
+        # Reset the players' boards
+        playerX = [0, 0, 0, 0, 0, 0, 0, 0, 0]
+        playerY = [0, 0, 0, 0, 0, 0, 0, 0, 0]
+
+        # Mark the moves on the board
+        for move in moves_X:
+            playerX[move] = 1
+        for move in moves_Y:
```
Comment (DurvaG8) [inline score_update_logic.py:33]: Can we do this in any other way? (Start a review comment)

File: score_update_logic.py (hunk 2 above)
Comment (harshitashyale) [reply score_update_logic.py:42]: I can try

**Comment by harshitashyale** [comment]
I'm not sure

**Comment by DurvaG8** [comment]
Do you see this Haley?

**Comment by DurvaG8** [comment]
Why is this so confusing

File: score_update_logic.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            playerY[move] = 1
+
+        score_X, score_Y = calculate_score(playerX, playerY)
+        print("Score for Player X:", score_X)
+        print("Score for Player Y:", score_Y)
+
```
Comment (DurvaG8) [inline score_update_logic.py:39]: Do I even know what is happening?

**Comment by hkkim97** [comment]
i see it

**Review by DurvaG8 — COMMENTED** [review]
do you see me?

**Comment by hkkim97** [comment]
> Do you see this Haley?

yes

File: score_update_logic.py (hunk 4 above)
Comment (harshitashyale) [reply score_update_logic.py:39]: Who even knows

File: score_update_logic.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+if __name__ == "__main__":
+    playerX = [0, 0, 0, 0, 0, 0, 0, 0, 0]
+    playerY = [0, 0, 0, 0, 0, 0, 0, 0, 0]
+    print("Welcome to Tic-Tac-Toe Score Calculation")
+    while True:
+        print("Enter the moves of players (0-8 for each player) separated by spaces (e.g., '0 1 2 4 7 8'):")
+        moves_X = list(map(int, input("Moves for X: ").split()))
+        moves_Y = list(map(int, input("Moves for Y: ").split()))
+
```
Comment (DurvaG8) [inline score_update_logic.py:25]: Let's start a new review

**Review by DurvaG8 — CHANGES_REQUESTED** [review]
hbiqnskqwms

**Comment by harshitashyale** [comment]
Jan,ns

File: score_update_logic.py [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+            score_Y += 1
+    return score_X, score_Y
+
+
+if __name__ == "__main__":
+    playerX = [0, 0, 0, 0, 0, 0, 0, 0, 0]
+    playerY = [0, 0, 0, 0, 0, 0, 0, 0, 0]
+    print("Welcome to Tic-Tac-Toe Score Calculation")
+    while True:
+        print("Enter the moves of players (0-8 for each player) separated by spaces (e.g., '0 1 2 4 7 8'):")
```
Comment (harshitashyale) [inline score_update_logic.py:22]: hi

**Comment by harshitashyale** [comment]
> How about if we try to redo it?

File: score_update_logic.py (hunk 5 above)
Comment (harshitashyale) [reply score_update_logic.py:25]: hi

**Pull request closed without merging**